### DISCUSSION TITLE
Where are hidden formulas which describe binding to Albumin ?

### DISCUSSION THREAD

**Discussion by Microbiotman** [Open-Systems-Pharmacology/Forum#1525] [Questions & Problems] [answered]
Hi all, 

I try to understand how distribution processes are implemented in MoBi and I was wondering where were hidden formulas which describe the binding to albumin in plasma ? Indeed, I could only find the albumin ontogeny factor which is just a parameter.  
For example, I'm looking for the formula which talks about the stoichiometry between albumin and my compound ? Does this formula exist ? 
Thank you very much for your help !

**Comment by StephanSchaller** [comment, marked as answer]
HI @Microbiotman , ALbumin binding is a static concept expressed within the parameter "fu" (fraction unbound in plasma). There is no actual binding process taking place.

You could implement this explicitly by defining a binding protein with expression according to the tissue distribution of Albumin, but this would neglect distribution kinetics / mass flow of Albumin-bound compound

To account for the latter, you have to implement Albumin as an actual PK.Sim compound in MoBi with synthesis (and degradation partially in the liver and through the endosomal system)
